galaxy_ng/tests/unit/api/test_api_ui_user_viewsets.py

```diff
@@ -372,6 +372,159 @@ def test_me_delete(self):
             response = client.delete(url, format="json")
             self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
+    def test_me_content_admin_permissions(self):
+        user = auth_models.User.objects.create(username="content_admin_user")
+        user.save()
+
+        group = self._create_group(
+            "",
+            "users_with_content_admin_permissions",
+            users=[user],
+            roles=["galaxy.content_admin"],
+        )
+        self.client.force_authenticate(user=user)
+
+        new_user_data = {
+            "username": "content_admin_user",
+            "first_name": "Tsrif",
+            "last_name": "Tsal",
+            "email": "email@email.com",
+            "groups": [{"id": group.id, "name": group.name}],
+        }
+
+        with self.settings(GALAXY_DEPLOYMENT_MODE=DeploymentMode.STANDALONE.value):
+            self._test_create_or_update(
+            self.client.put, self.me_url, new_user_data, status.HTTP_200_OK, user
+        )
+            client = APIClient(raise_request_exception=True)
+            client.force_authenticate(user=user)
+            response = self.client.get(self.me_url)
+            content_admin_permissions = LOCKED_ROLES["galaxy.content_admin"]["permissions"]
+            for i in content_admin_permissions:
+                self.assertEqual({response.data["model_permissions"][i]["has_model_permission"]}, {True})
```

Comment:
These asserts also need to check that the responses we expect to be false are also false. Here's you're just checking the ones we expect to be true, which means that this would also pass if the API just returned true for all the permissions all the time.

Reply:
also, there's no reason to put each condition in {}. You can either do `self.assertEqual(response.data["model_permissions"][i]["has_model_permission"], True)` or `self.asserTrue(response.data["model_permissions"][i]["has_model_permission"])`

Reply:
The trouble is that `self.assertEqual(response.data["model_permissions"][i]["has_model_permission"], True)` never works - it gives the AssertionError: {True} != True, hence the brackets. `self.assertTrue(response.data["model_permissions"][i]["has_model_permission"])` does work okay.

SImilarly, `self.assertFalse({response.data["model_permissions"][i]["has_model_permission"]})` has the AssertionError: {False} is not false, so I had to go back to `self.assertEqual - {False}`.